src/Microsoft.ML.SearchSpace/Parameter.cs

```diff
@@ -430,16 +430,28 @@ public override int GetHashCode()
             return thisJson.GetHashCode();
         }
 
+        /// <summary>
+        /// Determines wether two <see cref="Parameter"/> objects have the same value.
+        /// </summary>
+        /// <param name="x">The first parameter to compare.</param>
+        /// <param name="y">The second parameter to compare.</param>
+        /// <returns>true if the value of <paramref name="x" /> is the same as the value of <paramref name="y" />; otherwise, false.</returns>
         public bool Equals(Parameter x, Parameter y)
         {
             return x.GetHashCode() == y.GetHashCode();
         }
 
+        /// <summary>
+        /// <inheritdoc/>
```

Comment:
>inheritdoc [](http://example.com/codeflow?start=13&length=10)

should this be outside the summary tag? I think we need to fix all occurrence in this file.

Reply:
I was just copying all other instances from the same file.  I'll follow up with your suggestions.